locales/locales.php

```diff
@@ -1998,6 +1998,18 @@ public function __construct() {
 		$pcd->nplurals = 2;
 		$pcd->plural_expression = 'n > 1';
 
+		$pa_pk = new GP_Locale();
```

Comment:
It seems like you used `pnb` first for the variable name? I'd move this right next to the `Panjabi (India)` definition since the list is sorted by the variable name.